app/services/curation_concerns/curation_concern.rb

```diff
@@ -1,9 +1,20 @@
 module CurationConcerns
   module CurationConcern
-    def self.actor(curation_concern, *args)
+    # Returns the top-level actor on the stack
+    def self.actor(curation_concern, current_user, attributes)
+      AddToCollectionActor.new(curation_concern, current_user, attributes,
```

Comment:
Expectations on this API are a little odd. An Actor will take a subsequent actor AND call it? The subsequent actor can be either an array or a single instance (making the interface more than just sending a message.) I wonder if the solution here is to define a Pipeline with Pipes, and define an interface for a Pipe. Then the Pipeline's responsible for pushing everything through.